exchanges/btse/btse.go

```diff
@@ -518,7 +520,10 @@ func (b *BTSE) SendAuthenticatedHTTPRequest(ep exchange.URL, method, endpoint st
 		Verbose:       b.Verbose,
 		HTTPDebugging: b.HTTPDebugging,
 		HTTPRecording: b.HTTPRecording,
-		Endpoint:      f,
+	}
+
+	return b.SendPayload(context.Background(), f, func() (*request.Item, error) {
+		return item, nil
```

Comment:
This function also uses timestamp based signature and headers. I would advise inlining. BTSE already has rate limits